R/msaGaugeRRnonrep.R

```diff
@@ -169,6 +169,18 @@ msaGaugeRRnonrep <- function(jaspResults, dataset, options, ...) {
     ms <- anovaTable$MS
     f <- anovaTable$F.value
     p <- anovaTable$p.value
+    if (any(df < 1)) {
+      if (plotOnly) {
+        p <- ggplot2::ggplot() +
+          jaspGraphs::themeJaspRaw() +
+          jaspGraphs::geom_rangeframe()
+        return(p)
+      } else {
+        gaugeRRNonRepTable1$setError(gettextf("No degrees of freedom for %s.", anovaSources[which(df < 1)]))
```

Comment:
Can there only be at most one `df < 1`? Because I don't think the example below is intended?
```r
anovaSources <- letters[1:4]
df <- rep(0, 4)
gettextf("No degrees of freedom for %s.", anovaSources[which(df < 1)])
> [1] "No degrees of freedom for a." "No degrees of freedom for b." "No degrees of freedom for c."
[4] "No degrees of freedom for d."
```

Reply:
Good point